cibyl/sources/elastic/filter.py

```diff
@@ -0,0 +1,142 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+import argparse
+import logging
+from cibyl.sources.elastic.client import ElasticSearchClient
+from sys import exit
+
+LOG = logging.getLogger(__name__)
+
+
+class ElasticSearchOSPFilter:
```

Comment:
This seems to be the source itself, if yes, then I think we need to rename the file name and the class name as well so when users specify this:

type: elasticsearch_osp (or elasticsearchosp or es_osp, ....)

We will be able to import this class using source.type variable

Reply:
- I've modified the name `filter.py`  to `api.py`. Following the same case of the Zuul source.
- I've modified the name of the class from `ElasticSearchOSPFilter` to ElasticSearchOSP`.